src/js/utils/__tests__/ApplicationUtil-test.js

```diff
@@ -9,44 +9,60 @@ describe('ApplicationUtil', function () {
 
   describe('#invokeAfterPageLoad', function () {
 
+    beforeEach(function () {
+      // Clean up application timers.
+      jasmine.clock().uninstall();
```

Comment:
Yes, I know of this problem, and this workaround is a necessity, unfortunately.

I can't find the original discussion anymore :( But here is as far back I could trace it: https://github.com/dcos/dcos-ui/pull/313#discussion_r65678489